src/main/java/uk/gov/hmcts/darts/transcriptions/component/impl/TranscriberTranscriptsQueryImpl.java

```diff
@@ -260,11 +263,76 @@ public List<TranscriberViewSummary> getTranscriberTranscriptions(Integer userId)
                 ORDER BY
                     transcription_id desc
                 """,
-            new MapSqlParameterSource("usr_id", userId)
-                .addValue("rol_id", TRANSCRIBER.getId())
+            new MapSqlParameterSource(USR_ID, userId)
+                .addValue(ROL_ID, TRANSCRIBER.getId())
                 .addValue("current_date", LocalDate.now(clock).toString()),
             transcriberViewSummaryRowMapper
         );
     }
 
+    @Override
+    public List<Integer> getAuthorisedCourthouses(Integer userId, Integer roleId) {
+        return jdbcTemplate.queryForList(
+            """
+                select
```

Comment:
Does this query work? I tried running it in my local postgres and empty result set was returned for my user and rol_id=5. Perhaps I'm doing something wrong though as it's been a long day

Reply:
yes it works. can you try again with rol_id=4 (transcriber) let me know outcome